beta/package.json

```diff
@@ -35,6 +35,7 @@
     "react": "18.0.0-alpha-930c9e7ee-20211015",
     "react-collapsed": "3.1.0",
     "react-dom": "18.0.0-alpha-930c9e7ee-20211015",
+    "remark-slug": "^7.0.0",
```

Comment:
Do we need it in dependencies or devDependencies?  🤔

Reply:
moved to dev-deps!